gcp-bundle/LICENSE

```diff
@@ -220,100 +220,90 @@ License: Apache 2.0 - http://www.apache.org/licenses/LICENSE-2.0
 
 --------------------------------------------------------------------------------
 
-Group: com.google.api  Name: api-common  Version: 2.13.0
+Group: com.google.api  Name: api-common  Version: 2.42.1
 License: Apache-2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
 License: BSD-3-Clause - https://github.com/googleapis/api-common-java/blob/main/LICENSE
 
 --------------------------------------------------------------------------------
 
-Group: com.google.api  Name: gax  Version: 2.30.0
+Group: com.google.api  Name: gax  Version: 2.59.1
 License: Apache-2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
 License: BSD-3-Clause - https://github.com/googleapis/gax-java/blob/master/LICENSE
 
 --------------------------------------------------------------------------------
 
-Group: com.google.api  Name: gax-grpc  Version: 2.30.0
+Group: com.google.api  Name: gax-grpc  Version: 2.59.1
 License: Apache-2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
 License: BSD-3-Clause - https://github.com/googleapis/gax-java/blob/master/LICENSE
 
 --------------------------------------------------------------------------------
 
-Group: com.google.api  Name: gax-httpjson  Version: 2.30.0
+Group: com.google.api  Name: gax-httpjson  Version: 2.59.1
 License: Apache-2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
 License: BSD-3-Clause - https://github.com/googleapis/gax-java/blob/master/LICENSE
 
 --------------------------------------------------------------------------------
 
-Group: com.google.api-client  Name: google-api-client  Version: 2.2.0
+Group: com.google.api-client  Name: google-api-client  Version: 2.7.1
 Project URL: https://developers.google.com/api-client-library/java/
 License: The Apache Software License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
-Group: com.google.api.grpc  Name: gapic-google-cloud-storage-v2  Version: 2.22.5-alpha
+Group: com.google.api.grpc  Name: gapic-google-cloud-storage-v2  Version: 2.47.0
 License: Apache-2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
-Group: com.google.api.grpc  Name: grpc-google-cloud-storage-v2  Version: 2.22.5-alpha
-License: Apache-2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
-
---------------------------------------------------------------------------------
-
-Group: com.google.api.grpc  Name: proto-google-cloud-storage-v2  Version: 2.22.5-alpha
```

Comment:
I don't see this listed elsewhere. Are you sure it can be removed? The description says it is removing duplicates.

Reply:
Same with the one above.

Reply:
I mean that we have multiple `proto-google-cloud-storage-v2` entries in the `LICENSE`: 
* https://github.com/apache/iceberg/blob/main/gcp-bundle/LICENSE#L253
* https://github.com/apache/iceberg/blob/main/gcp-bundle/LICENSE#L258
* https://github.com/apache/iceberg/blob/main/gcp-bundle/LICENSE#L263

So I keep just one entry here.

Reply:
I don't think it's multiple, they are different packages technically?

gapic-google-cloud-storage-v2 vs grpc-google-cloud-storage-v2 vs proto-google-cloud-storage-v2 

I'll double check which packages we actually bundle code from

Reply:
That's right they are actually in the jar. I re-add it.

Reply:
I thought `gapic-google-cloud-storage-v2` includes the two others, but it's not the case.

Reply:
@jbonofre was this fixed? I still see that they are removed but I don't see different package names.

Reply:
@rdblue it should be ok now.